bitnami/minio/templates/distributed/statefulset.yaml

```
@@ -6,7 +6,7 @@ SPDX-License-Identifier: APACHE-2.0
 {{- if (eq .Values.mode "distributed") }}
 {{- $fullname := include "common.names.fullname" . }}
 {{- $headlessService := printf "%s-headless" (include "common.names.fullname" .) | trunc 63 }}
-{{- $releaseNamespace := .Release.Namespace }}
+{{- $releaseNamespace := (include "common.names.namespace" .) }}
```

Parenthesis not required:
```suggestion
{{- $releaseNamespace := include "common.names.namespace" . }}
```